RecoJets/JetAlgorithms/interface/HEPTopTaggerWrapperV2.h

```diff
@@ -0,0 +1,300 @@
+//  2011 Christopher Vermilion
+//
+//----------------------------------------------------------------------
+//  This file is free software; you can redistribute it and/or modify
+//  it under the terms of the GNU General Public License as published by
+//  the Free Software Foundation; either version 3 of the License, or
+//  (at your option) any later version.
```

Comment:
Since there was no comment from @davidlange6 @sextonkennedy to this, I'm assuming this message is OK.